src/dk/aau/cs/ds306e18/tournament/Main.java

```diff
@@ -36,6 +29,7 @@ public void start(Stage primaryStage) throws Exception {
 
     public static void main(String[] args) {
         System.out.println("Hej verden!");
```

Comment:
Not necessarily related to this PR, but I think we should remove this easter-egg. It's just confusing for non-Danish speakers.

Reply:
I have removed it, but it is usually not seen anyway because there is no console. (11f2e2e7fbd3275427)